test/service/test_jobs.py

```diff
@@ -121,7 +121,7 @@ def test_assigns_correct_geometry(self):
                          job.geometry)
 
     def test_assigns_correct_job_id(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.